app/src/main/java/ru/spbhse/pocketmagic/LoadActivity.java

```diff
@@ -0,0 +1,44 @@
+package ru.spbhse.pocketmagic;
+
+import android.content.Intent;
+import android.content.pm.ActivityInfo;
+import android.os.Bundle;
+import android.os.Handler;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.view.WindowManager;
+
+import ru.spbhse.pocketmagic.R;
+
+public class LoadActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+        setContentView(R.layout.activity_load);
+        getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN);
+
+        /* Wait 2 seconds before starting other activity. */
```

Comment:
Why? This is a type of comments you want to avoid. The code already says *what* it does. Comments should explain to the reader *why* it does what it does.

Reply:
Fixed